README.md

```diff
@@ -2,7 +2,7 @@
 
 # Fields
 
-A collection of commonly used fields implemented as custom Ecto types 
+A collection of commonly used fields implemented as custom Ecto types
 with the validation, sanitising and encryption/hashing. <br />
```

Comment:
I'm not sure but the `with the` sounds a bit weird to me, should it just
```suggestion
with validation, sanitising and encryption/hashing. <br />
```
but maybe it's fine

Reply:
Good catch. it's bad grammar. 👍